.travis.yml

```diff
@@ -1,9 +1,9 @@
 sudo: false
 language: node_js
 node_js:
-  - '4'
   - '6'
-  - '7'
+  - '8'
+  - '10'
```

Comment:
Can this be 8, 10, 12?

Reply:
I think so! I can make it happen. Can't imagine that anyone is still using 6 in our system.